app/models/miq_report_result.rb

```diff
@@ -71,7 +73,7 @@ def html_rows(options = {})
     end
     update_attribute(:last_accessed_on, Time.now.utc)
     purge_for_user
-    html_details.apply_legacy_finder_options(options.merge(:order => "id asc")).collect(&:data)
+    html_details.order("id asc").offset(options[:offset]).limit(options[:limit]).collect(&:data)
```

Comment:
Very good question.

We only ever pass in `:per_page`, and `:page`.
We actually pass in both every time but one, when we generate the pdf.